barte/models.py

```diff
@@ -0,0 +1,118 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional, List, Dict, Any
+
+@dataclass
+class Customer:
+    name: str
+    tax_id: str
+    email: str
+
+@dataclass
+class CardToken:
+    id: str
+    type: str
+    created_at: datetime
+    last_digits: str
+    holder_name: str
+    expiration_month: int
+    expiration_year: int
+    brand: str
+
+    @classmethod
+    def from_dict(cls, data: Dict[str, Any]) -> "CardToken":
+        if isinstance(data["created_at"], str):
+            data["created_at"] = datetime.fromisoformat(data["created_at"].replace("Z", "+00:00"))
```

Comment:
In Python 3.11 and newer, the `replace` method is unnecessary. Additionally, Python 3.10 is approaching its end of life. Is it worth support to support Python 3.10?